server/vertx-server/src/main/scala/sttp/tapir/server/vertx/streams/package.scala

```diff
@@ -21,7 +25,68 @@ package object streams {
       readStream: ReadStream[WebSocketFrame],
       pipe: streams.Pipe[REQ, RESP],
       o: WebSocketBodyOutput[streams.Pipe[REQ, RESP], REQ, RESP, _, VertxStreams]
-    ): ReadStream[WebSocketFrame] =
-      throw new UnsupportedOperationException()
+    ): ReadStream[WebSocketFrame] = {
+      val stream0 = optionallyContatenateFrames(readStream, o.concatenateFragmentedFrames)
+      val stream1 = optionallyIgnorePong(stream0, o.ignorePong)
+      val stream2 = optionallyAutoPing(stream1, o.autoPing)
+
+      val stream3 = new ReadStreamMapping(stream2, { (frame:WebSocketFrame) =>
+        o.requests.decode(frame) match {
+          case failure: DecodeResult.Failure =>
+            throw (new WebSocketFrameDecodeFailure(frame, failure))
+          case DecodeResult.Value(v) => v
+        }
+      })
+
+      new ReadStreamMapping(pipe(stream3), o.responses.encode)
+    }
+  }
+
+  private def optionallyContatenateFrames(rs: ReadStream[WebSocketFrame], doConcatenate: Boolean): ReadStream[WebSocketFrame] = {
+    //TODO implement this
+    rs
+  }
+
+  private def optionallyIgnorePong(rs: ReadStream[WebSocketFrame], ignore: Boolean): ReadStream[WebSocketFrame] = {
+    //TODO implement this
+      rs
+    }
+
+  private def optionallyAutoPing(rs:ReadStream[WebSocketFrame], autoPing: Option[(FiniteDuration, WebSocketFrame.Ping)]): ReadStream[WebSocketFrame] = {
+    //TODO implement this
+    rs
+  }
+}
```

Comment:
reproducing here [what's here for the others stream lib](https://github.com/softwaremill/tapir/blob/5a1044c154f9bd1b629dfc567096f55408d93da3/server/vertx-server/zio/src/main/scala/sttp/tapir/server/vertx/zio/streams/zio.scala#L171-L202) . Everything is working without the implementations, but would be nice to support those features in the futures.